lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```

Comment:
There are a few other places in the theme where we force things like `invert(1)`, if we want to make it an option should that be another PR as a feature rather than slotted into this fix PR?